README.md

```diff
@@ -1,26 +1,29 @@
-# Intro
+# Introduction
 
-Welcome to the start of a new adventure, here we will show you everything you need to do to setup a Minecraft Java server and host it! We will go through all the steps in as much detail as possible, and who knows? you might become the next hypixel!
+Welcome to the start of a new adventure! In this repository we will show you everything you need to do to setup a Minecraft Java Edition server! 
+We will go through the steps in as much detail as possible
+Who knows? You might become the next Hypixel!
 
 ## Tutorials
 
-### English
-
-#### Setup
+### Setup
 
 - [Finding what server jar to use](./tutorials/en_us/Finding%20what%20server%20jar%20to%20use.md)
 - [Setting up your minecraft server](./tutorials/en_us/Setting%20up%20your%20minecraft%20server.md)
 
 ## Contributing
 
-If you would like to Contribute just make a pull request with the changes and I'll review it
+If you would like to contribute, pleae make a pull request.
```

Comment:
Maybe add the link to the CONTRIBUTING.md and correct "pleae" to "please"

Reply:
Just accept HEAD when resolving conflicts later.